Source/PeleLM_derive.cpp

```diff
@@ -265,3 +351,44 @@ void pelelm_dergrdpz (const Box& bx, FArrayBox& derfab, int dcomp, int /*ncomp*/
 
 }
 
+
+
+//
+// Compute species concentrations C_n
+//
+
+void pelelm_derconcentration (const Box& bx, FArrayBox& derfab, int dcomp, int ncomp,
+                  const FArrayBox& datfab, const Geometry& /*geomdata*/,
+                  Real /*time*/, const int* /*bcrec*/, int /*level*/)
+
+{   
+    auto const in_dat = datfab.array();
+    auto       der = derfab.array();
+    int nspec_comp = in_dat.nComp() - 2;  //here we get back the correct number of species
+    
+    amrex::ParallelFor(bx, 
+    [=] AMREX_GPU_DEVICE (int i, int j, int k) noexcept
+    {   
+        amrex::Real Yt[nspec_comp], Ct[nspec_comp];
```

Comment:
I thought we couldn't have multiple declarations on the same line?

Reply:
I don't know, it works that way, but if you think this is a potential issue I have put everything on separated lines.

Reply:
Might as well leave it. Not sure why it works now but didn't before.  Maybe it has to do with whether you used the function or macro version of the launch.